k8/deploy/react-deploy.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+  namespace: team-promptheus
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      containers:
+      - name: frontend
+        image: ghcr.io/aet-devops25/team-promptheus-frontend:latest
+        ports:
+        - containerPort: 8080
```

Comment:
we likely want a service and igress to expose this to the outside world.
Just having a contaiener in the sandbox..